public/bootstrap-datepicker/css/bootstrap-datepicker3.css

```diff
@@ -0,0 +1,791 @@
+/*!
```

Comment:
@nmohamed just fixed it. now we don't have to actually store the JS and CSS locally!